pyproject.toml

```diff
@@ -66,6 +66,7 @@ tests = [
   "py-cpuinfo",
   "pytest",
   "pytest-cov",
+  "pytest-mock",
```

Comment:
I discovered this plugin/wrapper the other day for something else, its really handy.